go.mod

```diff
@@ -1,12 +1,25 @@
 module github.com/couchbase/gocb-opentelemetry
 
 require (
-	github.com/couchbase/gocb/v2 v2.4.0
-	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/otel v1.4.1
-	go.opentelemetry.io/otel/metric v0.27.0
-	go.opentelemetry.io/otel/sdk v1.4.1
-	go.opentelemetry.io/otel/trace v1.4.1
+	github.com/couchbase/gocb/v2 v2.6.3
+	github.com/stretchr/testify v1.8.3
+	go.opentelemetry.io/otel v1.16.0
+	go.opentelemetry.io/otel/metric v1.16.0
+	go.opentelemetry.io/otel/sdk v1.16.0
+	go.opentelemetry.io/otel/sdk/metric v0.39.0
+	go.opentelemetry.io/otel/trace v1.16.0
 )
 
-go 1.16
+require (
+	github.com/couchbase/gocbcore/v10 v10.2.3 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/go-logr/logr v1.2.4 // indirect
+	github.com/go-logr/stdr v1.2.2 // indirect
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/sys v0.8.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+go 1.20
```

Comment:
Does this have to be bumped to 1.20 or can it be left at 1.16?

Reply:
It requires go1.18 or later, Because opentelemtry is using generic types.